app-text/tesseract/tesseract-3.04.00-r4.ebuild

```diff
@@ -92,28 +103,41 @@ src_unpack() {
 		 -execdir sh -c 'cp -- "$0" "${S}/tessdata/${0%-*}"' '{}' ';' || die
 }
 
+src_prepare() {
+	epatch "${PATCHES[@]}"
+	eautoreconf
+
+	java-pkg-opt-2_src_prepare
+}
+
 src_configure() {
-	local myeconfargs=(
+	econf \
```

Comment:
Yes! It is very important nitpick! Moving configure args with all those nonsense backslashes is painful. Also it's impossible to add comments.